app/controllers/volunteers_controller.rb

```diff
@@ -9,7 +9,8 @@ def create
     @volunteer = Volunteer.new(volunteer_params)
     result = CreateVolunteer.new(volunteer: @volunteer).call
     if result.success
-      redirect_to(root_path, success: result.message)
+      VolunteerRegistrationMailer.with(volunteer: @volunteer).notify.deliver_now
```

Comment:
@ColinW520 Should I move the mailer call to methods instead? There will be at least one more going to the volunteer to confirm their form submission.

Reply:
This should be done in the service

Reply:
Ok, that's what I thought we discussed but I got an error the first time I tried and the Rails docs had it in the controller, but I guess they're just showing the most basic use case. I have it working from the service now. Thanks!